x-pack/plugins/security/public/nav_control/nav_control_service.tsx

```diff
@@ -49,6 +68,14 @@ export class SecurityNavControlService {
         }
       }
     );
+
+    return {
+      getUserMenuLinks$: () =>
+        this.userMenuLinks$.pipe(map(this.sortUserMenuLinks), takeUntil(this.stop$)),
+      addUserMenuLinks: (userMenuLink: UserMenuLink[]) => {
+        this.userMenuLinks$.next(userMenuLink);
+      },
```

Comment:
I don't think this was in the spirit of what @pgayvallet and I were asking for. `next` will still replace the previously added links with whatever is provided by the subsequent caller.

The reason we asked this to be changed to `addUserMenuLinks` was so that multiple plugins could register menu links without clobbering the links that other plugins may have added.

Reply:
Sorry for merging prematurely! I got a little too excited, and I thought I addressed all of your feedback. I misunderstood the ask as just a rename. I'll go ahead and revert the commit, and open a new PR.